boards/modalai/voxl2/default.px4board

```diff
@@ -24,5 +24,6 @@ CONFIG_MODULES_NAVIGATOR=y
 CONFIG_MODULES_COMMANDER=y
 CONFIG_MODULES_MAVLINK=y
 CONFIG_MODULES_CONTROL_ALLOCATOR=y
+CONFIG_MODULES_SIMULATION_PWM_OUT_SIM=y
```

Comment:
Does apps processor need this?

Reply:
Strangely I thought it wouldn't but when I do not include this in the apps as well and try to set the HIL parameters for HIL_ACT_FUNC1, etc... , I get that those parameters dont exist and it does not work.